float64 x_goal
float64 y_goal
float64 max_vel
---
uint8 ok
